lychee-bin/src/formatters/stats/compact.rs

```diff
@@ -52,7 +54,7 @@ impl Display for CompactResponseStats {
                 let mut sorted_suggestions: Vec<_> = suggestions.iter().collect();
                 sorted_suggestions.sort_by(|a, b| {
                     let (a, b) = (a.to_string().to_lowercase(), b.to_string().to_lowercase());
-                    human_sort::compare(&a, &b)
+                    cmp(&a, &b)
```

Comment:
There is [`std::cmp`](https://doc.rust-lang.org/std/cmp/index.html). Can we use the fully qualified name here to avoid the naming confusion?

```suggestion
                    numeric_sort::cmp(&a, &b)
```